test/home.test.js

```diff
@@ -0,0 +1,33 @@
+import home from '../src/Views/home';
+
+describe('firstSection function', () => {
+  const firstSection = home.firstSection();
+  it('Should return an HTML section tag with a class name of first-section', () => {
+    expect(firstSection.nodeName === 'SECTION' && firstSection.getAttribute('class')).toBe('col-12 col-md-3 d-flex flex-column justify-content-start text-white h-100vh first-section');
+  });
+});
```

Comment:
* [ ] Here the method the core result expected from the function is not tested, let us test the result it returns, in this case `section`.

Reply:
Its a function that returns a container so the core result here is a section that is empty with only two headers.
`   const section = it.is('section');
`
then returned as
`  return section;
`